docs/administration/csv-mappers.md

```diff
@@ -0,0 +1,72 @@
+# CSV Mappers
```

Comment:
We need to add the new capabilites for csv mapper in file capa#list-of-capabilities
The capa is : TAXIIAPI_SETCSVMAPPERS

Reply:
All good, thanks !